sktime/transformations/series/adi_cv.py

```diff
@@ -15,27 +16,27 @@ class ADICVTransformer(BaseTransformer):
     ``"smooth"``, ``"erratic"``, ``"intermittent"``, or ``"lumpy"``.
 
     The labels are based on the Average Demand Interval (ADI) and the Coefficient of
-    Variation (CV2) of the time series, using simple thresholding.
-    Optionally, the transformer can return the ADI and CV2 values as well.
+    Variation squared (CV^2) of the time series, using simple thresholding.
+    Optionally, the transformer can return the ADI and CV^2 values as well.
 
     Let :math:`x_t` be the value of the time series at times :math:`t = 1, 2, \dots, T`,
     and let :math:`N` be the number of non-zero values in the time series, i.e.,
     :math:`N = \text{card}\{t: x_t \neq 0\}`.
 
-    The ADI and CV2 are calculated as follows:
+    The ADI and CV^2 are calculated as follows:
 
-    1. Average Demand Interval (ADI): The average time period between
-      time periods with non-zero demands, mathematically defined as:
```

Comment:
it's actually non-zero, given the code. In-principle, this could be applied to series with negative values, too.